packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/toolbar/toolbar.css

```diff
@@ -99,6 +100,14 @@
 	@nest .ck-toolbar-container & {
 		border: 0;
 	}
+
+	& .ck.ck-button {
```

Comment:
If we move it I would consider moving the unit tests to `buttonview.js` file (if possible).

I have a feeling that we'll be also able to avoid creating the editor there to test this behavior.